website/content/en/highlights/2024-05-06-0-38-0-upgrade-guide.md

```diff
@@ -0,0 +1,57 @@
+---
+date: "2024-05-06"
+title: "0.38 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.38.0"
+authors: ["jszwedko"]
+release: "0.38.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.38.0 release includes a **breaking change**:
+
+1. [Vector now requires mandatory environment variables](#strict-env-vars)
+
+We cover them below to help you upgrade quickly:
```

Comment:
Should this line be after the deprecation list ?

Reply:
Good catch, thanks!